README.md

```diff
@@ -28,7 +28,8 @@ UPSTREAM_CHE_PATH=/path/to/upstream/che
 mvn -DwithoutKeycloak                          `# disable Keycloak support` \
     -Pfast                                     `# skip tests and other verifications` \
     -PmultiThread                              `# enable maven multi threading` \
-    -DlocalCheRepository=${UPSTREAM_CHE_PATH}  `# get already built upstream Che` \
+    -DlocalCheRepository=${UPSTREAM_CHE_PATH}  `# get already built upstream Che local folder` \ 
```

Comment:
@sunix I was wondering too why we need to add  `!checkout-base-che`. When I run the old command che wasn't checked out. Can you be more explicit? Is it when che branch isn't `openshift-connector`?